tket2-py/src/lib.rs

```diff
@@ -1,6 +1,4 @@
 //! Python bindings for TKET2.
-#![warn(missing_docs)]
```

Comment:
is this inherited now?

Reply:
Yes, that's the
```toml
[lints]
workspace = true
```
in Cargo.toml (since rust 1.74)